extensions/pipeline/setup.go

```diff
@@ -110,7 +111,7 @@ func UpdateEULA(adminClient *rancher.Client) error {
 
 	urlSetting.Value = fmt.Sprintf("https://%s", adminClient.RancherConfig.Host)
 
-	_, err = steveClient.SteveType("management.cattle.io.setting").Update(urlSettingResp, urlSetting)
+	_, err = steveClient.SteveType(stevetypes.Setting).Update(urlSettingResp, urlSetting)
```

Comment:
For posterity, are you sure there's no SteveSetting equivalent? If not, this is fine. Otherwise I would argue this needs to be stevetypes.ManagementSetting

Reply:
Not that I know of but I agree that ManagementSetting is much more clear. I will change it for clarity.